icepyx/core/query.py

```diff
@@ -329,12 +335,12 @@ def spatial_extent(self):
 
         """
 
-        if self.extent_type == "bounding_box":
-            return ("bounding box", self._spat_extent)
-        elif self.extent_type == "polygon":
+        if self._sp_extent.extent_type == "bounding_box":
+            return "bounding box", self._sp_extent.spatial_extent
```

Comment:
```suggestion
            return ("bounding box", self._sp_extent.spatial_extent)
```

pretty sure this needs to be a tuple